modules/installation-guide/partials/con_persistent-volume-configuration.adoc

```diff
@@ -5,62 +5,112 @@
 [id="storage-strategies-for-{prod-id-short}-workspaces_{context}"]
 = Storage strategies for {prod-id-short} workspaces
 
-Workspace Pods use Persistent Volume Claims (PVCs), which are bound to the physical Persistent Volumes (PVs) with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[ReadWriteOnce access mode]. It is possible to configure how the {prod-short} server uses PVCs for workspaces. The individual methods for this configuration are called PVC strategies:
+Storage strategy (PVC strategy):: A configurable method defining how {prod-short} workspaces uses persistent volume claims (PVCs). It defines the default data storage when storing in the workspace volumes data such as:
++
+* projects
+* workspace logs
+* additional Volumes defined by a user
 
-include::example${project-context}-pvc-strategies.adoc[]
+.Storage strategies comparison
+[width="100%",cols="1,1,1,1",options="header"]
+|===
+| Storage strategy name
+| `common`
+| `per-workspace`
+| `unique`
+
+| PVC count
```

Comment:
```suggestion
| PV count
```

Reply:
A PVC is the request for storage, a PV is a chunk of storage. PV count and PVC should be the same. But as an admin I probably want to know how many PVs I have to provision rather than the number of PVC that Che is going to issue. That's not critical but it may be worth reviewing this article usage of PVC vs PV.